docs/develop/docs.md

```diff
@@ -47,9 +47,37 @@ In a list of three or more items, the list item proceeding the word "and" should
 | One, two, three, and four.    | One, two, three and four.    |
 | Charles, Ada, and Alan.       | Charles, Ada and Alan.       |
 
+
+### Capitalization of Concepts and Tools
+
+Webrecorder has a number of common nouns that we use in our products. Examples include: archived items, crawl workflows, browser profiles, collections, and organizations. Because these are concepts and not specific instances of each concept, do not capitalize them unless they are at the start of a sentence.
+
+##### Example
+
+When starting a sentence:
+> Archived Items consist of one or more...
```

Comment:
Since it's not a proper noun the second word shouldn't be capitalized:
```suggestion
> Archived items consist of one or more...
```

Reply:
Alternatively we could just leave out this example to avoid confusion, since writers familiar with English will capitalize the first word.

Reply:
I specifically wanted to leave this in to distinguish between "Archived items" (start of sentence), "archived items" (mid sentence), and the "Archived Items page" (proper noun for pages).